src/main/resources/lang/dutch.lang

```diff
@@ -0,0 +1,190 @@
+	invalid folder: Kan de folder <grey>'<gold>%s<gold>'<red> niet in de scripts folder vinden!
+	reload:
+		warning line info: <gold><bold>Lijn %s:<gray> (%s)<reset>\n
+		error line info: <light red><bold>Lijn%s:<gray> (%s)<reset>\n
+		reloading: <gold>%s<reset> herladen...
+		reloaded: <lime>Succesvol herladen <gold>%s<lime>. <gray>(<gold>%2$sms<gray>)
+		error: <gold>%2$s <light red>error¦¦s¦ <light red>tegengekomen tijdens het herladen <gold>%1$s<light red>! <gray>(<gold>%3$sms<gray>)
+		script disabled: <gold>%s<reset> is momenteel uitgeschakeld. Gebruik <gray>/<gold>skript <cyan>enable <red>%s<reset> om het in te schakelen.
+		warning details: <yellow>    %s<reset>\n
+		error details: <light red>    %s<reset>\n
+		other details: <white>    %s<reset>\n
+		line details: <gold>    Lijn: <gray>%s<reset>\n <reset>
+
+		config, aliases and scripts: de configuratie, aliassen en alle scripts
+		scripts: alle scripts
+		main config: de hoofdconfiguratie
+		aliases: de aliassen
+		script: <gold>%s<reset>
+		scripts in folder: alle scripts in <gold>%s<reset>
+		x scripts in folder: <gold>%2$s <lime>script¦¦s¦ in <gold>%1$s<reset>
+		empty folder: <gold>%s<reset> bevat geen ingeschakelde scripts.
+	enable:
+		all:
+			enabling: Alle uitgeschakelde scripts inschakelen...
+			enabled: Alle eerder uitgeschakelde scripts ingeschakeld en geparseerd
+			error: <gold>%s <light red>error¦¦s¦ <light red>tegengekomen bij het parseren van uitgeschakelde scripts!
+			io error: <light red>Kon een of meer scripts niet laden - sommige scripts zijn mogelijk al hernoemd en zullen worden ingeschakeld wanneer de server opnieuw opstart: <gold>%s
+		single:
+			already enabled: <gold>%s<reset> is al ingeschakeld! gebruik <gray>/<gold>skript <cyan>reload <red>%s<reset> om het te herladen als het aangepast is.
+			enabling: <gold>%s<reset> inschakelen...
+			enabled: <gold>%s<reset> succesvol ingeschakeld en geparseerd.
+			error: <gold>%2$s <light red>error¦¦s¦ <light red>tegengekomen bij het parseren van <gold>%1$s<light red>!
+			io error: <light red>Kon <gold>%s <light red>niet inschakelen: <gold>%s
+		folder:
+			empty: <gold>%s<reset> bevat geen uitgeschakelde scripts.
+			enabling: <gold>%2$s<reset> script¦¦s¦ in <gold>%1$s<reset> inschakelen...
+			enabled: <gold>%2$s<reset> eerder uitgeschakelde scripts in <gold>%1$s<reset> succesvol ingeschakeld en geparseerd.
+			error: <gold>%2$s <light red>error¦¦s¦ <light red>tegengekomen bij het parseren van scripts in <gold>%1$s<light red>!
+			io error: <light red>Fout bij het inschakelen van een of meer scripts in <gold>%s <light red>(sommige scripts worden mogelijk ingeschakeld wanneer de server opnieuw opstart): <gold>%s
+	disable:
+		all:
+			disabled: Alle scripts met succes uitgeschakeld.
+			io error: <light red>Kon een of meer scripts niet hernoemen - sommige scripts zijn mogelijk al hernoemd en worden uitgeschakeld wanneer de server opnieuw wordt opgestart: <gold>%s
```

Comment:
same for using `kon [...]` here

Reply:
Also same explanation for why I believe `kon` should be used here

Reply:
Changed it to a different sentence structure